scripts/rabbitmq-integration-test.sh

```diff
@@ -53,6 +53,7 @@ function fatal() {
   message=$2
 
   echo "ERROR: $message"
+  cat setupdb.log
```

Comment:
It gets the failing log output into the CI output so I think it's a good thing to keep